Tests/Amr/Advection_AmrLevel/Source/AmrLevelAdv.cpp

```diff
@@ -285,53 +309,88 @@ AmrLevelAdv::advance (Real time,
     }
 
 #ifdef AMREX_USE_OMP
-#pragma omp parallel
+#pragma omp parallel if (Gpu::notInLaunchRegion())
 #endif
     {
         FArrayBox flux[BL_SPACEDIM], uface[BL_SPACEDIM];
-
         for (MFIter mfi(S_new, true); mfi.isValid(); ++mfi)
         {
+            // Set up tileboxes and nodal tileboxes
             const Box& bx = mfi.tilebox();
+            GpuArray<Box,BL_SPACEDIM> nbx;
+            AMREX_D_TERM(nbx[0] = mfi.nodaltilebox(0);,
+                         nbx[1] = mfi.nodaltilebox(1);,
+                         nbx[2] = mfi.nodaltilebox(2));
 
+            // Grab fab pointers from state multifabs
             const FArrayBox& statein = Sborder[mfi];
             FArrayBox& stateout      =   S_new[mfi];
 
-            // Allocate fabs for fluxes and Godunov velocities.
+            // Add elixir for output state fab
+            Elixir steli = stateout.elixir();
```

Comment:
```suggestion
```
This should be removed.  Elixir is supposed to be used on temporary Fabs, not on `S_new`.